streamalert/artifact_extractor/artifact_extractor.py

```diff
@@ -112,10 +112,26 @@ def artifacts(self):
             # Return immediately if can not identify source_type. a.k.a do not extract artifacts.
             return artifacts
 
+        # normalized information in the record will be similar to
+        # {
+        #   'record': {
+        #       'region': 'us-east-1',
+        #       'detail': {
+        #           'awsRegion': 'us-west-2'
+        #       }
+        #   },
+        #   'normalization': {
+        #       'region': {
+        #           'values': ['us-east-1', 'us-west-2']
+        #           'function': 'AWS region'
```

Comment:
The structure of this object will not work. If you have 2 `region` type values with different `function` values it will conflict.

```
"streamalert:normalization": {
  "ip_address": [
    {
      "values": ["4.3.2.1"],
      "function": "outbound_connection_destination"
    },
    {
      "values": ["2.2.2.2", "4.4.4.4"],
      "function": "dns_lookup"
    }
    }
  ]
}
```